### PR TITLE
docs: add link to invitation issue

### PR DESCRIPTION
**Pull request by schmelto**
think it would be a good thing to add a link to the "Invite me to the Github Organization" issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![Discord](https://img.shields.io/discord/699608417039286293?style=flat-square)](https://discord.gg/jZQs6Wu)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-Raise an issue to join the EddieHub GitHub community.
+Raise an [issue to join the EddieHub GitHub community](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization).
```
Comment (vinzvinci) [inline README.md:5]: Which is better, `issue to join the EddieHub GitHub community` or `issue to join the EddieHub GitHub organisation`?

Comment (schmelto) [reply README.md:5]: > Which is better, `issue to join the EddieHub GitHub community` or `issue to join the EddieHub GitHub organisation`?

Think organization is the more accurate term 🤔

**Comment by schmelto** [comment]
Is it called `Organization` or `Organisation`

Our Action uses the option with `s`
see [here](https://github.com/EddieHubCommunity/support/pull/1379#pullrequestreview-643253671)
image


GitHub with `z`
see [here](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations)
image

I've found an article about this 
> The only difference is that organization is the sole spelling used in American English, while both terms are common in British English.

see [here](https://writingexplained.org/organisation-vs-organization-difference#:~:text=Organisation%20is%20an%20alternative%20spelling,are%20common%20in%20British%20English.)

**Comment by vinzvinci** [comment]
Both are correct:
`'s' is the correct letter in the UK and then 'z' in the US.
`
Let's use **organisation** for Eddie :).

**Comment by schmelto** [comment]
> Let's use **organisation** for Eddie :).

👍🏼

**Review by vinzvinci — APPROVED** [review]
Looks good to me. I think it much better now than the other one.

File: README.md (hunk 1 above)
Comment (kaiwalyakoparkar) [reply README.md]: According to me "Community " sounds more helpful and friendly

**Review by tash2020 — APPROVED** [review]
LGTM

**Review by eddiejaoude — APPROVED** [review]
Looks great 👍

**Pull request merged**